gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -576,6 +576,7 @@
      * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#valuemap-step" target="_blank">Reference Documentation - ValueMap Step</a>
      * @since 3.0.0-incubating
      */
+    @Deprecated
```

Comment:
needs javadoc "@deprecated As of release 3.4.0, replaced by {@link ...}" also needed below in the `__` class.